### PR TITLE
gitignore add tailwind rules for ignore

### PR DESCRIPTION
**Pull request by Yhugzee**

### CODE REVIEW

File: frontend/.gitignore [hunk 1]
```diff
@@ -13,6 +13,10 @@ dist-ssr
 *.local
 .env
 
+tailwind.config.js
+vite.config.ts
+postcss.config.js
```
Comment (rocambille) [inline frontend/.gitignore:18]: this should not be ignored :)

File: frontend/src/App.css [hunk 2]
```diff
@@ -1,56 +1,22 @@
...
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+  background-image: url("marvel-vs-dc-4k-iu.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 100vh;
```
Comment (rocambille) [inline frontend/src/App.css:14]: prefer min-height

File: frontend/src/components/Menu.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+function Menu() {
+  return (
+    <div>
```
Comment (rocambille) [inline frontend/src/components/Menu.jsx:5]: div should not have titles, header do

File: frontend/src/pages/Home.jsx [hunk 4]
```diff
@@ -1,36 +1,11 @@
...
-        <a
-          className="App-link"
-          href="https://vitejs.dev/guide/features.html"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Vite Docs
-        </a>
-      </p>
+    <header>
```
Comment (rocambille) [inline frontend/src/pages/Home.jsx:5]: use header in Menu ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/.gitignore (hunk 1 above)
Comment (Yhugzee) [reply frontend/.gitignore:18]: Okay ^^

File: frontend/src/App.css (hunk 2 above)
Comment (Yhugzee) [reply frontend/src/App.css:14]: ✅

File: frontend/src/components/Menu.jsx (hunk 3 above)
Comment (Yhugzee) [reply frontend/src/components/Menu.jsx:5]: ✅

File: frontend/src/pages/Home.jsx (hunk 4 above)
Comment (Yhugzee) [reply frontend/src/pages/Home.jsx:5]: ✅

**Comment by rocambille** [comment]
conflicts to resolve

**Pull request merged**